src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,137 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getFilm } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+const imdbUrl = 'https://www.imdb.com/title/';
+
+export const FindMovie = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [errorVisibility, seterrorVisibility] = useState(false);
+  const [filmsInList, setFilmInList] = useState([
+    'tt1375666', 'tt0314331', 'tt0319262', 'tt3748528', 'tt0457939',
+  ]);
```

Comment:
```suggestion
```
It's a bad practice to manually add some values. You could use the method `some` in your `addMovie` function with the `movies` array to check by imdb id if the film is already present